ai-decision-services/ai-ad-dpp-engine/introduction/introduction.md

```diff
@@ -2,46 +2,78 @@
 
 ## About this Workshop
 
-This introduction covers the complete "parent" workshop. Use this text to set up the story for the workshop. Be engaging - what will the learner get from spending their time on this workshop?
+In this lab, we are going to offer a detailed hands-on onboarding guide for users who are interested in our automated end to end **Anomaly Detection** data preprocessing and training/inference for workflow.
 
-Estimated Workshop Time: -- hours -- minutes (This estimate is for the entire workshop - it is the sum of the estimates provided for each of the labs included in the workshop.)
+## Objectives
+This solution will allow you to configure a training pipeline and an inference pipeline in your tenancy, along with ingestion and preprocessing of your data. Input sizes could range from a few rows to several tens of millions of rows, and 1 column to several thousands of columns.
 
-*You may add an option video, using this format: [](youtube:YouTube video id)*
 
-  [](youtube:zNKxJjkq0Pw)
+## Workflow introduction
 
-### Objectives
+![workflow](../attachments/workflow.png)
 
-*List objectives for the workshop*
+A brief description of the whole picture:
```

Comment:
I prefer the original wording: "In this workshop, you will learn how to:". Or even better, in passive voice
 "This workshop demonstrates how to:"

Reply:
This is mentioned at the session **About this Workshop**. Here is actually an explanation on what the whole processing workflow looks like.